sysinit/coqargs.ml

```diff
@@ -382,6 +384,10 @@ let parse_args ~usage ~init arglist : t * string list =
       let native_output_dir = next () in
       { oval with config = { oval.config with native_output_dir } }
 
+    |"-output-dir" | "-output-directory" ->
+      let dir = CUnix.canonical_dir (next ()) in
```

Comment:
I think that messing with the directory the user provides is not a good idea. Why are you doing this?

For example, error messages won't contain the original directory, and the extra resolution may cause other issues.

IMHO this is a blocker for this PR.

Reply:
The motivation is to be robust over `Cd`: it is more intuitive that a relative directory given on the command line is resolved relatively to the command line than dynamically depending on which `Cd`s have done so far.

But the question of reporting an error is a good question. So maybe, what can be done is to kept the pair of both the relative output directory and of the directory in which the command has been launched??